packages/react-native/ReactAndroid/src/main/java/com/facebook/react/bridge/Inspector.kt

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.bridge
+
+import com.facebook.common.logging.FLog
+import com.facebook.jni.HybridData
+import com.facebook.proguard.annotations.DoNotStrip
+import com.facebook.react.common.ReactConstants
+  
+@DoNotStrip
+public class Inspector private constructor(private val hybridData: HybridData) {
+ 
+    public companion object {
+        init {
+            BridgeSoLoader.staticInit()
+        }
+
+        @JvmStatic
+        public fun getPages(): List<Page> {
+            return try {
+                instance().getPagesNative().toList()
+            } catch (e: UnsatisfiedLinkError) {
+                FLog.e(ReactConstants.TAG, "Inspector doesn't work in open source yet", e)
+                emptyList()
+            }
+        }
+
+        @JvmStatic
+        public fun connect(pageId: Int, remote: RemoteConnection): LocalConnection {
+            return try {
+                val local = instance().connectNative(pageId, remote)
+                local ?: throw IllegalStateException("Can't open failed connection")
```

Comment:
I think you can simplify this as:

```suggestion
                instance().connectNative(pageId, remote) ?: throw IllegalStateException("Can't open failed connection")
```